src/Spice86.Core/Emulator/Devices/Sound/PCSpeaker/PcSpeaker.cs

```diff
@@ -204,39 +185,45 @@ public override void WriteByte(ushort port, byte value) {
             _loggerService.Verbose("PC Speaker set value {PCSpeakerValue}", ConvertUtils.ToHex8(value));
         }
 
-        switch (port)
-        {
-            case 0x61:
-            {
-                SpeakerControl oldValue = _controlRegister;
-                _controlRegister = (SpeakerControl)value;
-                if ((oldValue & SpeakerControl.SpeakerOn) != 0 && (_controlRegister & SpeakerControl.SpeakerOn) == 0) {
+        switch (port) {
```

Comment:
Perhaps the ReadByte and WriteByte should handle the port check in the same way?

Reply:
I guess you are talking about switch vs if.
It was a remain from the old pc speaker code that also implemented a very basic counter on port 42.
I uniformized both methods